cfme/tests/test_filters.py

```diff
@@ -0,0 +1,142 @@
+from cfme.web_ui import search
+from cfme.web_ui.cfme_exception import assert_no_cfme_exception
+
+import fauxfactory
+import pytest
+from cfme.login import login_admin, logout
+from utils.blockers import BZ, GH
+from utils import version
+
+if version.current_version() >= '5.6.2.0':
```

Comment:
version picking should not be at module level, since it forces SSSH connection on module load, which is eek.